src/components/SearchToolbar/index.js

```diff
@@ -0,0 +1,28 @@
+import "./SearchToolbar.scss";
+
+export const SearchToolbar = ({ city, country, setData, theme }) => {
+	const handleCitySearch = (event) => {
+		if (event.key === "Enter") {
+			setData(prev => ({
+				...prev,
+				city: event.target.value,
+			}))
+		}
+	}
+	const switchTheme = (event) => {
```

Comment:
so. let's move your theme state in App.js. create in utils/constants/theme.js const THEMES = {black: 'black', white: 'white'} and use it in App state. so here u will not need an event as arg and check your class name